dht-node/.env.dist

```diff
@@ -0,0 +1,26 @@
+CONFIG_VERSION=v14.2022-11-22
+
+# Server
+PORT=5000
+GRAPHIQL=false
+
+# Database
+DB_HOST=localhost
+DB_PORT=3306
+DB_USER=root
+DB_PASSWORD=
+DB_DATABASE=gradido_community
+TYPEORM_LOGGING_RELATIVE_PATH=typeorm.dht-node.log
+
+# EventProtocol
+EVENT_PROTOCOL_DISABLED=false
+
+# SET LOG LEVEL AS NEEDED IN YOUR .ENV
+# POSSIBLE VALUES: all | trace | debug | info | warn | error | fatal
+# LOG_LEVEL=info
+
+# Federation
+# if you set the value of FEDERATION_DHT_TOPIC, the DHT hyperswarm will start to announce and listen
+# on an hash created from this topic
+FEDERATION_DHT_TOPIC=GRADIDO_HUB
+# FEDERATION_DHT_SEED=64ebcb0e3ad547848fef4197c6e2332f 
```

Comment:
how is it ensured for deployment on production, that a dht-seed is set? bare-metal or command-line/system-parameter?